### DISCUSSION TITLE
Why is MayaUSD-0.23 compiled for Maya2022.3 and not for Maya2022.4 (as in previous releases) ?

### DISCUSSION THREAD

**Discussion by VVVSLAVAkz** [Autodesk/maya-usd#2996] [General] [answered]
Why is MayaUSD-0.23 compiled for Maya2022.**3** and not for Maya2022.**4** (as in previous releases) ?
Is this true, or is it a typo?

**Comment by santosg87** [comment, marked as answer]
hi @VVVSLAVAkz that would be a typo indeed!

with that said, though, all of the changes that are available for Maya 2022 had no changes affect compatibility with the previosu Maya version, so the plugin should be compatible with 2022.3 as well as 2022.4 :)

**Reply by VVVSLAVAkz** [reply]
No, that's not a typo!
Maya USD-0.23 for Maya2022 is actually compiled with MAYA API 2022.3 , not 2022.4!

<a href="https://smms.app/image/WqyQCbAZO6KLNGP" target="_blank"><img src="https://s2.loli.net/2023/04/06/WqyQCbAZO6KLNGP.png" alt="usd_023_maya_2022_4.png"></a>
